pkgs/applications/office/libreoffice/default.nix

```diff
@@ -92,9 +90,6 @@ in (mkDrv rec {
     libs = list:
       lib.concatMapStringsSep " " (e: "-L${lib.getLib e}/lib") list;
   in ''
-    substituteInPlace shell/source/unix/exec/shellexec.cxx \
-      --replace /usr/bin/xdg-open ${if kdeIntegration then "kde-open5" else "xdg-open"}
```

Comment:
Doesn't it make sense to replace even if short path is used?

Reply:
No idea.

Reply:
Sounds like a good idea to minimise unrelated change.

Reply:
Sorry, I just don't know what you are referring to :)

Reply:
Fully dropping this will change behaviour of KDE-integration builds; conditionally replacing `xdg-open` with `kde-open5` should keep it.